api/src/services/script-runner.service.ts

```diff
@@ -53,6 +54,46 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @param jurisdictionIdDTO id containing the jurisdiction id we are creating the new community type for
+   * @returns successDTO
+   * @description creates a new reserved community type. Reserved community types also need translations added
+   */
+  async createNewReservedCommunityType(
+    req: ExpressRequest,
+    jurisdictionIdDTO: IdDTO,
+  ): Promise<SuccessDTO> {
+    // script runner standard start up
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      'Senior Veterans Community Type',
+      requestingUser,
+    );
+
+    // create new veteran community type
+    await this.prisma.reservedCommunityTypes.create({
```

Comment:
I don't think I would have anticipated that we would commit this, are we just somehow gonna run it pointed at prod?

Reply:
Hey @emilyjablonski I wrote up some documentation on how script runner works. But the abridged version is:

we write and commit scripts, deploy them, and then run them via API calls to that environment's API. 

so in this case, we will write a script that gets deployed to prod, then we'll run the script using prod's api

hopefully that makes some sense, but if not maybe the tech doc explains it better

Reply:
It's all good for this to not be blocking, but something about committing this feels a little unusual, maybe we can chat when we go over the tech doc!